SConstruct

```diff
@@ -340,24 +340,24 @@ if env["platform"] == "":
 
 if env["platform"] == "osx":
     # Deprecated alias kept for compatibility.
-    print_warning('Platform "osx" has been renamed to "macos" in Godot 4. Building for platform "macos".')
```

Comment:
Blazium was not what changed the platform names, Godot 4 was accurate.

Reply:
will fix it in the squashed commit